repos/system_upgrade/el7toel8/libraries/tests/test_isccfg.py

```diff
@@ -116,6 +116,8 @@
 };
 """)
 
+views_empty = isccfg.MockConfig("")
+
```

Comment:
I would suggest also ``MockConfig('options { include "/dev/null"; };')`` testing, so we test also empty include file. I would rename it to ``config_empty`` variable, it contains no view.